### PR TITLE
Added support for iOS in `graph.py`

### PR DESCRIPTION
**Pull request by AnujSaha0111**
Update display method in graph.py

### CODE REVIEW

File: matplotlib_visual/graph.py [hunk 1]
```diff
@@ -4,30 +4,54 @@
...
-
-        # Windows
         elif "win32" in sys.platform:
             subprocess.run(
                 ["C:\\Windows\\System32\\clip.exe"],
                 input=text.strip().encode(),
                 check=True,
             )
-
+        elif "darwin" in sys.platform:
```
Comment (this-is-yaash) [inline matplotlib_visual/graph.py:47]: I see that you added support for iOS, and I really appreciate your efforts! However, that was not the intended outcome. It was actually already fixed while I was rebasing with the previous branches. I’ll rename this PR and issue to reflect the addition of iOS support.

**Comment by AnujSaha0111** [comment]
Was it merged under the new updated issue

**Comment by this-is-yaash** [comment]
Hey @AnujSaha0111 
I noticed that you are a new contributor! I want to give you some feedback on what you did well:  
You created a separate branch in your **forked repository** (`AnujSaha0111:Update-display-method-in-graph.py`) and submitted the PR from that branch. This is excellent practice, as making changes in the main branch is not recommended.  

Great work! It's good to go.
I will merge it shortly!

**Review by this-is-yaash — APPROVED** [review]

**Comment by this-is-yaash** [comment]
It seems, we still have some conflicts that are preventing me from merging the PR.
Could you please take a look at it and let me know?

**Comment by AnujSaha0111** [comment]
Updated it, check it once

**Comment by this-is-yaash** [comment]
Can you please remove the comments and make another PR?
I think newly added comments causing conflicts with main branch.

**Comment by AnujSaha0111** [comment]
Removed the comments and sent a new PR

**Comment by this-is-yaash** [comment]
Please make sure you pushed the PR!
I didn't receive any PR req.

**Comment by AnujSaha0111** [comment]
CAN YOU PLEASE CHECK AGAIN

**Comment by this-is-yaash** [comment]
I think you forgot to add **`commit`** and **`push`**.

**Comment by AnujSaha0111** [comment]
Did it in different PR

**Comment by this-is-yaash** [comment]
> Did it in different PR

but that PR, doesn't have any changes you've made. 
It's just a PR without any changes!
Could you please look into it and let me know?

**Pull request closed without merging**